src/Files.App/Assets/FilesOpenDialog/Files.App.Launcher.exe.sha256

```diff
@@ -1 +1 @@
-9f09add6084f2b30451d91864a573cc4cd0bf728a1df9bfc87fd1b7027487c60
+881d6caef54765251a7e9c88df9a090c5dd939b17140c199d3d08259d5ced961
```

Comment:
Can you revert this change?

Reply:
Done. Reverted changes.

Reply:
The update to Files.App.Launcher.exe should also be reverted.

Reply:
Done. Thanks for review